config/default.js

```diff
@@ -0,0 +1,168 @@
+module.exports = {
+	GitterIRCBridges: [
+		{
+			gitter: {
+				token: process.env.NXB_GITTER_TOKEN,
+				channel: process.env.NXB_GITTER_CHANNEL,
+				commands: {
+					
+					list: {
+						not_elevated: true,
+						callback: function(user, args, gitter, irc) {
+							var users = irc.getUsers();
+							var msg = "";
+							if (users.length <= 1) {
+								msg = "[GitterBridge] There are no users online.";
+							} else {
+								for (var i = 0; i < users.length; i++) {
+									if (users[i] === irc.getUsername()) {
+										users.splice(i, 1);
+									}
+								}
+								msg = "[GitterBridge] There are " + users.length + " User(s) online: " + users.join(", ");
+							}
+							gitter.say(msg);
+						}
+					},
+					whois: {
+						not_elevated: true,
+						callback: function(user, args, gitter, irc) {
+							irc.whois(args[0], function(username, whoisinfo) {
+								gitter.say("[GitterBridge] Whois for " + username + ":\n```\n" + JSON.stringify(whoisinfo, null, "\t") + "\n```");
+							}.bind(this, user));
+						}
+					},
+					xcloud: {
+						not_elevated: true,
+						callback: function(user, args, gitter, irc) {
+							var msg = "XCloud Migration Guide: http://portal.new-xkit.com/assets/migration/xcloud-migration.html";
```

Comment:
okay this technically is a "config" but ideally this would be a property of the object I guess, not a local variable.